### PR TITLE
Move demo app

### PR DESCRIPTION
**Pull request by mohyour**
Moved web demo app from [asset-tracking-js](https://github.com/ably/ably-asset-tracking-js/tree/main/examples/subscribing-example-app)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "deliveryapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (QuintinWillison) [inline package.json:11]: We're using `Apache-2.0`.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "deliveryapp",
```
Comment (QuintinWillison) [inline package.json:2]: This should align with our conformed terminology. See: https://github.com/ably/asset-tracking-backend-demo#delivery-lifecycle

Perhaps:

```suggestion
  "name": "Ably Asset Tracking Web Customer Demo",
```

File: server.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const express = require('express');
```
Comment (QuintinWillison) [inline server.js:1]: This is not required. We have [a dedicated backend service](https://github.com/ably/asset-tracking-backend-demo) for this suite of demos.

Therefore, there is no reason why this Web demo should not be created as a static site.

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  root: true,
+  extends: "eslint:recommended",
```
Comment (QuintinWillison) [inline .eslintrc.js:3]: Can we please consider conforming to the ESLint rules established in other Ably repositories, ideally AirBnB per [ably-common](https://github.com/ably/ably-common/blob/main/.eslintrc.js) and [the backend](https://github.com/ably/asset-tracking-backend-demo/blob/main/functions/.eslintrc.js).

**Review by QuintinWillison — CHANGES_REQUESTED** [review]
My hope was that the work done on this demo would be a fresh start and greenfield. My worry with the 'move' that's being done here (I think it's actually a 'copy', right? 😉) is that we're bringing in a load of source code that was right for the context of that original example/test-harness but brings in too much baggage for this fresh demo, given [the wider context](https://github.com/ably/asset-tracking-demos/blob/main/app-requirements.md).

I certainly don't want to slow the work down by insisting that all of my observations get actioned on now, but I would like to at least see issues created for all of the observations within my review, so that they don't get forgotten to be addressed in subsequent PRs. That assumes, of course, that this 'copy' approach is agreed upon by all. 🤔

File: package.json (hunk 2 above)
Comment (QuintinWillison) [reply package.json:2]: Sorry, I just realised when working on something unrelated, that this name is not valid for use with `npm`.

We need to consider whether we should scope it (e.g. prefix with `@ably/`).

So, perhaps `@ably/asset-tracking-web-customer-demo`. What do you think, @owenpearson @Peter-Maguire @stmoreau?

File: server.js (hunk 3 above)
Comment (mohyour) [reply server.js:1]: @QuintinWillison the library loads the static files generated in the `dist` folder after builing [asset-tracking-js](https://github.com/ably/ably-asset-tracking-js). I updated the readme to include that information. I'm not exactly sure if that covers what you mean. If not, do you mind explaining further?

**Comment by mohyour** [comment]
Closing this because it's no longer needed.

**Pull request closed without merging**